### ISSUE TITLE
iframe support

### ISSUE THREAD

**Issue by wiiittttt** [tuvistavie/atomic-chrome#12]
It looks like the framework for this was added as a result of #11. 

I think there just needs to be another method added?

`normalizeIframe(elem) {
  return normalizeFrame(elem);
}`

**Comment by pixelchutes** [comment]
:+1: Yes, please!

**Comment by danhper** [comment]
Thanks for reporting, this should be fixed in 0.2.3.

**Comment by pixelchutes** [comment]
@tuvistavie ~~sorry to pester, but when testing `0.2.3` (`<textarea>` in `<frame>`), Atomic Chrome is opening up _two_ Atom tabs...~~

~~I don't believe this happened in `0.2.2`.~~

Oddly, it does not seem to work for a texteditor like CodeMirror (inside `<frame>`), but does work for CodeMirror without `<frame>` involved.

**UPDATE:**

~~The two Atom tabs seems to be happening even when `<frame>` is not involved. I'm going to attempt~~ Restarting Chrome `51.0.2693.2 dev (64-bit)` and Atom `1.6.1` seemed to resolve the duplication issue, just not sure why CodeMirror doesn't work inside of a frame?

console log:

> Uncaught (in promise) TypeError: Cannot read property 'on' of undefined(…) injected.js:1

``` javascript
value: function(e) {
    this.editor.on("change", this.wrapSilence(e))
           //  ^^ here
}
```

**Comment by danhper** [comment]
@pixelchutes Thanks for the catch!
I was using the `window` object instead of the iframe `window` object,
causing the error when working inside in iframe.
I just published a fix as 0.2.4, please let me know how it goes!

**Comment by pixelchutes** [comment]
@tuvistavie Confirmed, thank you! :beers:
